PlainSql.Migrations.Generator/PlainSql.Migrations.Generator.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Version>1.0</Version>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
@Nagelfar I'm only targeting net6.0 at the moment. Is this enough?

Reply:
You could also include 7.0, IMHO.
The `PlainSql.Migrations.Tool` is targeting older versions to not break any existing users, which is probably not the case for this one.
(Once `netcoreapp3.1` & `net5.0` start to cause any troubles I'd also phase them out because the .NET version is out of support anyway)